internal/contributor-info/releasing.md

```
@@ -196,6 +199,252 @@ RELEASE_CI_EVALUATE_HEAD=true bundle exec rake "release[17.0.0.rc.10]"
+seam does not change the ordinary blocking ShakaPerf waiter or its historical display-URL fallback.
+Conflicting duplicate run identities or equal ordering keys block independent of API order; only canonically identical
+duplicates collapse. A deterministically newer ordered success may supersede older ordered failures after normal
+runtime-equivalent evidence verification, while a newer ordered failure remains blocking. Reusable successful pre-runs
+remain valid, and only fully formed canonically unrelated targets are ignored without poisoning the lane. Same-version
+retry discovery must complete successfully before the task can
+prove that accelerated history is absent; API, pagination, permission, or parse failure therefore blocks
+that retry as unknown. Repository and selected-tracker discovery read chronological 100-comment pages
+incrementally, validate string bodies, positive unique comment IDs, canonical repository issue URLs, parseable string
+creation and update timestamps, and nondecreasing creation times, and retain only comments containing the explicit hidden
+machine-marker opener
+with the literal ASCII opener `<!-- react-on-rails-accelerated-rc `, including its single trailing space. A plain-text
```

markdownlint-cli2 (0.23.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)